docs/_guides/databases.md

```diff
@@ -100,29 +101,18 @@ The most important bits of information are:
 * `doc_count`: the number of undeleted documents in the database
 * `db_name`: the name of the database
 
-{% include anchor.html title="Debugging" hash="debugging" %}
-
-When you create a PouchDB database, there are many ways to debug and inspect it.
-
-### PouchDB Inspector
 
-PouchDB Inspector is an add-on for Chrome that allows you to inspect your local databases.
-
-* [Download PouchDB Inspector for Chrome](https://chrome.google.com/webstore/detail/pouchdb-inspector/hbhhpaojmpfimakffndmpmpndcmonkfa)
-
-{% include img.html src="pouchdb_inspector.png" alt="PouchDB Inspector in Chrome" %}
```

Comment:
I think this image should probably be deleted too right?

Reply:
👍 Roger that. Will do this now.